fusor-ember-cli/app/mixins/openshift-mixin.js

```diff
@@ -69,7 +69,8 @@ export default Ember.Mixin.create(NeedsDeploymentMixin, {
   substractCfme: Ember.computed.not('ignoreCfme'),
 
   diskAvailableMinusCfme: Ember.computed("deployment.openshift_available_disk", "cfmeDisk", function() {
-    return this.get("deployment.openshift_available_disk") - this.get("cfmeDisk");
+    const rawDisk = this.get("deployment.openshift_available_disk") - this.get("cfmeDisk");
+    return Math.round(rawDisk * 100) / 100;
```

Comment:
Instead of `Math.round()`, do you want to use `Math.ceil()` or `Math.floor()`? I'm not sure it matters.